### PR TITLE
Add codecov to measure code coverage across repo

### PR DESCRIPTION
**Pull request by Paulinakhew**
### WHY are these changes introduced?
- need a way to see code coverage

### WHAT is this pull request doing?
- implement codecov

### Checklist

- [ ] I have updated the CHANGELOG (if applicable)
- [ ] I have followed the [Shopify Python](https://github.com/Shopify/shopify_python) guide

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -21,8 +24,10 @@ jobs:
...
           python -m pip install --upgrade pip
-          pip install pytest mock
+          pip install pytest mock codecov
           python setup.py install 
       - name: Run Tests
-        run: python -m pytest -v
-
+        run: |
+          coverage run -m pytest -v
+      - name: Upload to codecov.io
```
Comment (andyw8) [inline .github/workflows/build.yml:32]: Perhaps we can use https://github.com/codecov/codecov-action for this?

**Comment by Paulinakhew** [comment]
@andyw8 do you have any idea why it's failing? 😢

**Comment by andyw8** [comment]
Is it perhaps because we need to run `python setup.py install` before `pytest --cov=./ --cov-report=xml`?

**Comment by Paulinakhew** [comment]
> Is it perhaps because we need to run `python setup.py install` before `pytest --cov=./ --cov-report=xml`?

😀 yes thank you that fixed it

**Review by andyw8 — APPROVED** [review]

**Pull request merged**